open-metadata-implementation/common-services/repository-handler/src/main/java/org/odpi/openmetadata/commonservices/repositoryhandler/RepositoryHandler.java

```diff
@@ -2097,6 +2266,51 @@ public void createRelationship(String                  userId,
         }
     }
 
+    /**
+     * Create a relationship from an external source between two entities.
+     *
+     * @param userId calling user
+     * @param externalSourceGUID unique identifier (guid) for the external source.
+     * @param externalSourceName unique name for the external source.
+     * @param relationshipTypeGUID unique identifier of the relationship's type
+     * @param end1GUID entity to store at end 1
+     * @param end2GUID entity to store at end 2
+     * @param relationshipProperties properties for the relationship
+     * @param methodName name of calling method
+     *
+     * @throws PropertyServerException problem accessing property server
+     * @throws UserNotAuthorizedException security access problem
+     */
+    public void createExternalRelationship(String                  userId,
+                                           String                  externalSourceGUID,
+                                           String                  externalSourceName,
+                                           String                  relationshipTypeGUID,
+                                           String                  end1GUID,
+                                           String                  end2GUID,
+                                           InstanceProperties      relationshipProperties,
+                                           String                  methodName) throws UserNotAuthorizedException,
+            PropertyServerException
+    {
+        try
+        {
+            metadataCollection.addExternalRelationship(userId,
```

Comment:
Wrong order for the parameters when calling addExternal. The signature for addExternalRelationship is  public Relationship addExternalRelationship(String               userId,
                                                String               relationshipTypeGUID,
                                                String               externalSourceGUID,
                                                String               externalSourceName,
                                                InstanceProperties   initialProperties,
                                                String               entityOneGUID,
                                                String               entityTwoGUID,
                                                InstanceStatus       initialStatus)